llvm/include/llvm/IR/Use.h

```diff
@@ -85,10 +84,8 @@ class Use {
   Use **Prev = nullptr;
   User *Parent = nullptr;
 
-  inline void addToList(unsigned &Count);
-  inline void addToList(Use *&List);
-  inline void removeFromList(unsigned &Count);
-  inline void removeFromList(Use *&List);
+  inline void addToList(Use **List);
+  inline void removeFromList();
```

Comment:
The previous PR moved the method away from here -- should move them back again? I think it makes more sense to define them here than in Value.h.

Reply:
That would require moving them out of line from the incomplete Value